platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/HtmlToSpansParser.kt

```diff
@@ -329,17 +335,16 @@ internal class HtmlToSpansParser(
         // If the link is not editable, tag all but the first character of the anchor text with
         // ExtraCharacterSpans. These characters will then be taken into account when translating
         // between editor and composer model indices (see [EditorIndexMapper]).
-        val isContentEditable = !last.span.contentEditable
+        val isContentEditable = last.span.data.containsKey("contenteditable")
```

Comment:
The presence of `contenteditable` doesn't imply the value is true so I think we'd need to check the value of the attribute here. It may actually be possible to remove this and replace it with an `isMention` check, but I'm not 100% sure.